docs/tutorial-code/7-min-max-timeslice-constraints/2-max-constraint/settings.toml

```diff
@@ -17,6 +17,7 @@ tolerance_unmet_demand = -0.1
 quantity = "prices"
 sink = "aggregate"
 filename = "{cwd}/{default_output_dir}/MCA{Quantity}.csv"
+index = false
```

Comment:
I think the `index` key is ultimately bundled up with other arguments passed to `to_csv`. If I'm right, this seems to be the only place where we do that: https://github.com/EnergySystemsModellingLab/MUSE_OS/blob/36f8c778312e2969d4bfc7730646335fba161a91/src/muse/outputs/sinks.py#L246

Reply:
So adding something like this might work:

```py
if not "index" in params:
    params["index"] = False
```